engines/native/sandbox.go

```diff
@@ -125,6 +137,21 @@ func fetchContext(context, destdir string) error {
 	if err = os.Chmod(filename, 0700); err != nil {
 		return fmt.Errorf("Error setting file '%s' permissions: %v", filename, err)
 	}
+	u, err := osuser.Lookup(user.Name())
```

Comment:
Especially, since you might want to do ACL kind of stuff on windows..

If we move it to `system` then it's very clear that native engine has no platform dependent code..